src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds.UnitTests/ServiceTests/GivenALearnerRefundProcessor/AndThereArePreviousPaymentsForTheMonth.cs

```diff
@@ -0,0 +1,373 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.Payments.DCFS.Domain;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Services;
+using SFA.DAS.ProviderPayments.Calc.Refunds.UnitTests.Utilities;
+using SFA.DAS.ProviderPayments.Calc.Refunds.UnitTests.Utilities.Extensions;
+using SFA.DAS.ProviderPayments.Calc.Refunds.UnitTests.Utilities.TestHelpers;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.UnitTests.ServiceTests.GivenALearnerRefundProcessor
+{
+    [TestFixture]
+    public class AndThereArePreviousPaymentsForTheMonth
```

Comment:
I really like this layout, can we as a sub team agree a standard going forward on how we write unit tests. We have several different styles floating around. One question if the Folder is GivenA.... then shouldn't each class be When.... and the sub classes be And....